src/types/coreBuildService.ts

```diff
@@ -20,9 +26,9 @@ export type IntegrationTestScenarioSpec = {
   environment?: Environment;
   params?: Param[];
   resolverRef?: {
+    resourceKind: string;
```

Comment:
since it's value can be only pipeline or PipelineRun derive the type from ResourceKind. something like
`typeof keyof ResourceKind`